cves/CVE-2015-6826.yml

```diff
@@ -355,8 +415,11 @@ CWE_instructions: |
 
   Just the number here is fine. No need for name or CWE prefix. If more than one
   apply here, then choose the best one and mention the others in CWE_note.
-CWE:
-CWE_note:
+CWE: 457
+CWE_note: |
```

Comment:
Do you agree with this classification?

Reply:
I do not agree with the CVE's CWE-20 classification. I will make that clearer.

Reply:
Update added